kimchi/src/lagrange_basis_evaluations.rs

```diff
@@ -2,18 +2,85 @@ use ark_ff::{batch_inversion_and_mul, FftField};
 use ark_poly::{EvaluationDomain, Evaluations, Radix2EvaluationDomain as D};
 use rayon::prelude::*;
 
-/// The evaluations of all normalized lagrange basis polynomials at a given
-/// point. Can be used to evaluate an `Evaluations` form polynomial at that point.
+/// Evaluations of all normalized lagrange basis polynomials at a given point.
+/// Can be used to evaluate an `Evaluations` form polynomial at that point.
+///
+/// The Lagrange basis for polynomials of degree <= d over a domain
+/// {ω_0,...,ω_d} is the set of (d+1) polynomials ${l_0,...,l_d}$ of degree d
```

Comment:
Note that LaTeX notations like `$` do not work in the doc. \` has to be used, like `{l_0, ..., l_d}`. It is pure Markdown that is accepted. See the generated doc by
```
RUSTDOCFLAGS="-D warnings" cargo doc --all-features --no-deps
```
and open
```
firefox target/doc/kimchi/index.html
```